### ISSUE TITLE
[A11Y] Unable to reach `Switch` during AddProfile

### ISSUE THREAD

**Issue by rt4914** [oppia/oppia-android#3904]
Unable to reach `Switch` during AddProfile

https://user-images.githubusercontent.com/9396084/136251645-73ca510f-d4a3-4aca-aeca-9a55c3082016.mp4
